### ISSUE TITLE
Regression in  SDnetDecoderBlock

### ISSUE THREAD

**Issue by glgerard** [shayansiddiqui/nn-common-modules#6]
In  the forward method of SDnetDecoderBlock, the call to self.unpool can fail if out_block is None.
Code should be changed as such

        if out_block is not None:
            unpool = self.unpool(input, indices, out_block.shape)
            concat = torch.cat((out_block, unpool), dim=1)
        else:
            unpool = self.unpool(input, indices)
            concat = unpool

**Comment by viethoang303** [comment]
I also got this problem and I can' t run the code
